features/app/components/AbstractApp.js

```diff
@@ -107,12 +121,18 @@ export class AbstractApp extends Component {
      */
     _openURL(url) {
         let room = this._getRoomFromUrlString(url);
+        let currentRoom =
+            this.props.store.getState()['features/base/conference'].room;
 
         // TODO Kostiantyn Tsaregradskyi: We should probably detect if user is
         // currently in a conference and ask her if she wants to close the
         // current conference and start a new one with the new room name.
 
-        this.props.store.dispatch(setRoom(room));
+        // XXX Prevents dispatching unnecessary SET_ROOM actions, which happened
+        // in web version.
```

Comment:
For now it's just about dispatching same action twice. I don't see if any re-renders will happen because of this for now.